app/scripts/remove_extra_dummy_events_in_sessions.py

```diff
@@ -0,0 +1,69 @@
+from pymongo import MongoClient
+import os
+
+# test session["_id"] in prod: 633646b0639829dad5a896b4
+
+if __name__ == "__main__":
+
+    if "MONGO_AUTH_CREDENTIALS" not in os.environ:
+        from dotenv import load_dotenv
+
+        load_dotenv("../.env.local")
+    client = MongoClient(os.getenv("MONGO_AUTH_CREDENTIALS"))
+    session_collection = client.quiz.sessions
+
+    # Iterate through all documents with an 'events' key
+    for session in session_collection.find(
+        {"events": {"$exists": True}, "$expr": {"$gt": [{"$size": "$events"}, 1000]}}
```

Comment:
why are we only filtering where size of events array is more than 1000?

Reply:
`session_collection.find({"events": {"$exists": True}})` I'll change it to this. Large event arrays will definitely contain dummy-events, so I filtered for one of those sessions to check how script is running.